examples/rsp-cra-18/src/ToDoItems.tsx

```diff
@@ -0,0 +1,41 @@
+import {CheckboxGroup, Checkbox, Flex} from '@adobe/react-spectrum'
+
+function TodoItems(props: { list : {id: number; task: string}[]; 
+                            handleList: any; 
+                            updateCompleted: any}) {
+
+    function removeItem(id: number, task: string){    
+        let index = getIndex(props.list, task);
+
+        props.updateCompleted(props.list[index].task); 
+        props.handleList(props.list.filter(item => item.id !== id));
+    }
+
+    function getIndex(list : {id: number; task: string}[], task: string){
```

Comment:
Are the ids of the todo items unique? If so, you can probably search skip checking the task property and just remove the item in the list with the matching id